buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -46,6 +46,8 @@ object Versions {
     val dokka = "0.9.17"
     val koin = "2.0.1"
     val jsonAssert = "1.5.0"
+    val ktor = "1.2.3"
```

Comment:
Hey, I cannot find any usage of ktor. Do we need it?

Reply:
This is a requirement for adam. I can separate it in a different PR

Reply:
Wait, adam is separate library. If you define this dependency in adam i see no reason to define it here too

Reply:
fixed